internal/manager/task_stash_box_tag.go

```diff
@@ -135,6 +169,7 @@ func (t *StashBoxPerformerTagTask) stashBoxPerformerTag(ctx context.Context) {
 			if performer.Measurements != nil && !excluded["measurements"] {
 				partial.Measurements = models.NewOptionalString(*performer.Measurements)
 			}
+			//TODO: This seems incorrect, is there a reason it's opposite?
 			if excluded["name"] && performer.Name != nil {
 				partial.Name = models.NewOptionalString(*performer.Name)
 			}
```

Comment:
This seems deliberate to have opposite logic for the excluded check, but I couldn't figure out what.
I did not replicate it for studio.